cli.js

```diff
@@ -233,28 +269,31 @@ print constraints
         Filename of the compiled circuit file generated by circom.
 
         Default: circuit.json
-        `)
-    .alias("c", "circuit")
-    .alias("pk", "provingkey")
-    .alias("vk", "verificationkey")
-    .alias("w", "witness")
-    .alias("p", "proof")
-    .alias("i", "input")
-    .alias("pub", "public")
-    .alias("v", "verifier")
-    .alias("lo", "logoutput")
-    .alias("lg", "logget")
-    .alias("ls", "logset")
-    .alias("lt", "logtrigger")
-    .help("h")
-    .alias("h", "help")
-
-    .epilogue(`Copyright (C) 2018  0kims association
+
+
+Options:
+  --version   Show version number                                      [boolean]
+  -h, --help  Show help                                                [boolean]
+
+Copyright (C) 2018  0kims association
     This program comes with ABSOLUTELY NO WARRANTY;
     This is free software, and you are welcome to redistribute it
     under certain conditions; see the COPYING file in the official
-    repo directory at  https://github.com/iden3/circom `)
-    .argv;
+    repo directory at  https://github.com/iden3/circom`, 
+    { full: 'circuit', short: 'c' }, 
+    { full: 'provingkey', short: 'pk' }, 
+    { full: 'verificationkey', short: 'vk' },
+    { full: 'witness', short: 'w' },
+    { full: 'proof', short: 'p' },
+    { full: 'input', short: 'i' },
+    { full: 'public', short: 'pub' },
+    { full: 'verifier', short: 'v' },
+    { full: 'logoutput', short: 'lo' },
+    { full: 'logget', short: 'lg' },
+    { full: 'logset', short: 'ls' },
+    { full: 'logtrigger', short: 'lt' },
+    { full: 'protocol' },
```

Comment:
protocol command is added, and help is removed. Is this intentional?

Reply:
Being able to select protocol is generally useful, though it doesn't have any affect on the way TC uses these libraries.

Help is still present, it is just special cased up on line 56